docs/file-reference.md

```diff
@@ -37,104 +40,135 @@ volumes:
 
-Version of OpenCompose format.
+The current version of the OpenCompose format
+
+## Section: Services
+
+```yaml
+services:
+- <Service>
+  ...
+
+- <Service>
+  ...
+```
+
+`services` is main section that lists all the services. 
 
-## services
-`services` is main section that lists all the services that this OpenCompose file describes.
-`services` has to be array of [ServiceSpec](#servicespec).
+`services` is an array of [Service](#service).
+
+### Service
 
-### ServiceSpec
 Describes one service. Service can be composed of one or multiple containers which are scheduled
 together and can communicate using localhost. (Containers in the same service share network namespace.)
 
 Each service has name and list of the containers, while `replicas` can be optional.
 
-```yml
-# <ServiceSpec>
-  name: foo
-  replicas: 4
+_Example:_
+
+```yaml
+services:
+- name: foobar
+  replicas: 3
   labels:
     foo_label: bar_label
   containers:
-  - <ContainerSpec>
+  - <Container>
   emptyDirVolumes:
-  - <EmptyDirVolumeSpec>
+  - <EmptyDirVolume>
 ```
 
 #### name
-| type | required |
+
+| Type | Required |
 |------|----------|
 |string|    yes   |
 
 Name of the service.
 
 #### replicas
-| type    | required |
+
+| Type    | Required |
 |---------|----------|
 | integer |    no    |
 
 Number of desired pods of this particluar service. This is an optional field. The valid value can only be a positive number.
 
 #### labels
-| type    | required |
+
+| Type    | Required |
 |---------|----------|
 | map with string keys and string values |    no    |
 
 Desired labels to be applied to the resulting Kubernetes objects from the service.
 
-### containers
-| type                                    | required |
+#### containers
+
+| Type                                    | Required |
 |-----------------------------------------|----------|
-|array of [ContainerSpec](#containerspec) |    yes   |
+|array of [Container](#container) |    yes   |
 
-Each item in [containers](#containers) ([ContainerSpec](#containerspec)) defines one container.
+Each item in [containers](#containers) ([Container](#container)) defines one container.
 All the containers in container array for given service will be in same Pod.
 
-### emptyDirVolumes
-| type                                              | required |
+#### emptyDirVolumes
+
+| Type                                              | Required |
 |---------------------------------------------------|----------|
 |array of [EmptyDirVolumeSpec](#emptydirvolumespec) |    no    |
 
 Each item in [emptyDirVolumes](#emptydirvolumes) ([EmptyDirVolumeSpec](#emptydirvolumespec)) defines a EmptyDir volume.
 These volumes will be shared among the containers in the service.
 
-### ContainerSpec
-```yml
-# <ContainerSpec>
-  image: foo/bar:tag
-  env:
-  - name: VARIABLE
-    value: value
-  command:
-    - /foo/bar
-  args:
-    - "-foo"
-  ports:
-    - <PortSpec>
-  mounts:
-    - <MountSpec>
+### Container
+
+```yaml
+services:
```

Comment:
@kadel A bit confused, I believe that line 138 example (what you comment on here), goes over every possible value of <Container>, having the full example on the top gives users a quick reference / being able to quickly go over and iterate through each value.

Putting "-services" and "-name:foobar" shows WHERE `-container` goes.